contracts/ts/__tests__/MACI.test.ts

```diff
@@ -113,12 +112,15 @@ describe('MACI', () => {
     let mpContract
     let tallyContract
     let pollId: number
+    let user: ethers.Signer
 
     describe('Deployment', () => {
         beforeAll(async () => {
             signer = await getDefaultSigner()
             const r = await deployTestContracts(
                 initialVoiceCreditBalance,
+                STATE_TREE_DEPTH,
+                true
```

Comment:
Why pass `true` here?

Reply:
oh basically those utils functions like deployTestContract or deployContract have an optional parameter called "quiet", passing true so it doesn't print unnecessary logs while running the tests